sdk/attestation/azure-security-attestation/pom.xml

```diff
@@ -0,0 +1,116 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+    <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+  </parent>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-security-attestation</artifactId>
+  <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-sdk-template;current} -->
```

Comment:
This will have to change to:
`<!-- {x-version-update;com.azure:azure-security-attestation;current} -->` 
and an entry will be added to our version_client.txt:
https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/version_client.txt#L87